lib/matplotlib/backend_bases.py

```diff
@@ -2693,6 +2693,10 @@ def set_message(self, s):
 
     def back(self, *args):
         """Move back up the view lim stack."""
+        if args:
+            cbook.warn_deprecated(
```

Comment:
This actually breaks (deprecation-warning) gtk3agg forward/backward/home buttons, because they connect callbacks which pass `event` to the method.  Haven't checked other backends but you should have a look too.

Reply:
I tend to leave `*args` in then and just document that it's for convenience but does nothing. What do you think?

Reply:
That's fine too.